### PR TITLE
[T2A5][F11 - A04]ChiangWengKiat 

### PR DESCRIPTION
**Pull request by wengkiat**
- Made the find command case insensitive
- Created a sort method to sort address book

### CODE REVIEW

**Comment by wengkiat** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -348,7 +348,7 @@ private static void loadDataFromStorage() {
     public static String executeCommand(String userInputString) {
         final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
         final String commandType = commandTypeAndParams[0];
-        final String commandArgs = commandTypeAndParams[1];
+        final String commandArgs = commandTypeAndParams[1].toLowerCase(); // to make the find function case insensitive
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:356]: that's all it takes: turning the input keyword to lowercase as long as you do the same normalisation when retrieving the data from the addressbook for comparison. But is this a good place to make this change? is it obvious to someone looking at your entire codebase the first time?

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -362,14 +367,28 @@ public static String executeCommand(String userInputString) {
...
+        	
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
-
-    /**
+    //sorting method
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:379]: bad header comment!

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -362,14 +367,28 @@ public static String executeCommand(String userInputString) {
...
+    private static String sortAddressBook() {
+    	Collections.sort(allPersons.subList(0, allPersons.size()), new Comparator<String[]>() {
+    		public int compare(String[] person2, String[] person1) {
+    			return person2[PERSON_DATA_INDEX_NAME].compareToIgnoreCase(person1[PERSON_DATA_INDEX_NAME]);
+    		}
+    	});
+    	savePersonsToFile(allPersons, storageFilePath);
+    	executeListAllPersonsInAddressBook();
+		return MESSAGE_SORT_SUCCESSFUL;
+	}
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:389]: indentation not according to coding style. use autoformat after every change. Add test cases for sort.

**Pull request closed without merging**